CHANGELOG.md

```diff
@@ -6,9 +6,13 @@ See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpC
 2.1.x (unreleased)
 ------------------
 
-* Upgraded phpunit to 5.7 / 6. If you use anything from the FOS\HttpCache\Test
-  namespace you need to update your project to use phpunit 6 (or 5.7, if you
-  are using PHP 5.6).
+* [Testing] Upgraded phpunit to 5.7 / 6. If you use anything from the
+  `FOS\HttpCache\Test` namespace you need to update your project to use
+  PHPUnit 6 (or 5.7, if you are using PHP 5.6).
+* [Symfony HttpCache] Added a `TaggableStore` for tag based invalidation with 
```

Comment:
This needs to be updated.

Reply:
i deleted the wrong half of this :-( thanks for spotting, fixed